README.md

```
@@ -1,149 +1,303 @@
+| `/packs` | Browse and manage your packs |
+| `/manage` | Manage your existing sticker packs |
+| `/sync` | Import an existing Telegram pack |
+| `/help` | Show help information |
+| `/about` | About Stix Magic |
+| `/cancel` | Cancel the current operation |
+
+---
+
+## Features
+
+### Draft Vault
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)